contracts/market/Market.sol

```diff
@@ -504,6 +504,8 @@ contract Market is IMarket, OwnableUpgradeSafe, Proxiable {
      * After the market is closed, anyone can trigger tokens to be swept to the owner
      */
     function recoverTokens(IERC20 token) public override {
+        require(address(token) != address(0x0), "Invalid address");
```

Comment:
Same thing here. If supplying token is not correct it'll revert anyhow, no need to checks. Not a big deal here, since this method is not called frequently

Reply:
good point, removed